packages/gestalt/src/Callout.js

```diff
@@ -193,7 +225,7 @@ export default function Callout({
 
 Callout.propTypes = {
   // $FlowFixMe[signature-verification-failure] flow 0.135.0 upgrade
-  dismissButton: PropTypes.exact({
+  dismissButton: PropTypes.shape({
```

Comment:
Curious why you needed to update from exact to shape?

Reply:
I actually did this because @AlbertCarreras asked me to make the same change on Upsell. I'm not sure the reasoning behind it though

https://github.com/pinterest/gestalt/pull/1298#discussion_r535920164

Reply:
mea culpa. Indeed, .shape is far more used than .exact so I thought for consistency. however, in many cases we should use .exact rather than .shape. 

@norrise120 , I apologize, We should leave `.exact`.

Reply:
Updated this and the other PR to be `.exact`. Thank you both!!

Reply:
Awesome, thank you!